src/storage/types.rs

```diff
@@ -80,7 +80,7 @@ impl MvccInfo {
 }
 
 /// Represents the status of a transaction.
-#[derive(PartialEq, Debug)]
+#[derive(PartialEq, Eq, Debug)]
```

Comment:
Why need this?

Reply:
It’s the derive_partial_eq_without_eq lint.

Reply:
Interesting. Will derive useless trait slow down compilation?

Reply:
I think it would: https://github.com/rust-lang/rust/issues/80118, I'll see if I can disable it.

Reply:
Updated and removed a bunch of unneeded `Eq`s.